serving/operator/deploy/resources/knative-serving-0.11.1.yaml

```diff
@@ -1053,21 +1183,18 @@ data:
 
     # When operating in a stable mode, the autoscaler operates on the
     # average concurrency over the stable window.
+    # Stable window must be in whole seconds.
     stable-window: "60s"
 
     # When observed average concurrency during the panic window reaches
     # panic-threshold-percentage the target concurrency, the autoscaler
     # enters panic mode. When operating in panic mode, the autoscaler
     # scales on the average concurrency over the panic window which is
     # panic-window-percentage of the stable-window.
+    # When computing the panic window it will be rounded to the closest
+    # whole second.
     panic-window-percentage: "10.0"
 
-    # Absolute panic window duration.
-    # Deprecated in favor of panic-window-percentage.
-    # Existing revisions will continue to scale based on panic-window
-    # but new revisions will default to panic-window-percentage.
-    panic-window: "6s"
```

Comment:
This is the kind of thing that the operator could deal with: checking for a `panic-window` key in the CR's config map-of-maps and either converting to some `panic-window-percentage` or logging a warning. Probably not worth doing for this, but good to keep in mind for future updates.